src/components/clusterConfiguration/FormatDiskWarning.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Text } from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { global_warning_color_100 as warningColor } from '@patternfly/react-tokens';
+
+const FormatDiskWarning = () => (
+  <Text component="p">
+    <ExclamationTriangleIcon className="status-icon" color={warningColor.value} size="sm" />
+    &nbsp;Note that all disks will be formatted when the installation starts.
```

Comment:
We don't format all disks. We are deleting the file system on bootable disks.

Reply:
So, a more accurate text should be: "Note that all bootable disks will be formatted when the installation starts"?

Reply:
`Note that bootable disks will be formatted when the installation starts.` is what merged in this PR, but adding `all` to that sentence seems like a good tweak for extra clarity. I've created #291 if you agree @rollandf.